em_examples/UXO_TEM_Widget.py

```diff
@@ -11,6 +11,7 @@
 from mpl_toolkits.mplot3d import Axes3D
 from mpl_toolkits.mplot3d.art3d import Poly3DCollection
 from scipy import sparse as spar
+from cvxopt import solvers,matrix
```

Comment:
@dccowan: can `scipy.optimize` (https://docs.scipy.org/doc/scipy-0.14.0/reference/tutorial/optimize.html) satisfy your needs here? It is a much more standard scientific python package and would keep the dependencies of em_examples more lightweight (which is preferable from the maintenance point of view)

Reply:
Tried it. It can't do what I want. This package was the best solution I found.